src/views/login.hbs

```diff
@@ -1,4 +1,12 @@
-<div> 
-    Hello from login
-</div>
-
+<form id="loginForm" class="loginForm">
```

Comment:
@AhmedAl-Almi You might not need to have the `class` set on this form.  Since you gave it an id, you could use that id in your css to style the form (and in the js to find the form element and get it's FormData).  It's not hurting anything to have the class either, it's just a little redundant. 🤷‍♂️